crates/cargo-contract/src/cmd/info.rs

```diff
@@ -71,25 +72,14 @@ pub struct InfoCommand {
 
 impl InfoCommand {
     pub async fn run(&self) -> Result<(), ErrorVariant> {
-        let client =
-            OnlineClient::<DefaultConfig>::from_url(url_to_string(&self.url)).await?;
+        let rpc = RpcClient::from_url(url_to_string(&self.url)).await?;
+        let client = OnlineClient::<DefaultConfig>::from_rpc_client(rpc.clone()).await?;
 
         // All flag applied
         if self.all {
             // 1000 is max allowed value
             const MAX_COUNT: u32 = 1000;
```

Comment:
MAX_COUNT should be removed. It is not needed, as we do not use the response paging anymore.

Reply:
Done :+1: